pkgs/development/libraries/libosinfo/default.nix

```diff
@@ -21,6 +21,16 @@ stdenv.mkDerivation rec {
 
   patches = [
     ./osinfo-db-data-dir.patch
+    (fetchpatch {
```

Comment:
Can you link to something external for details on the cve's

Maybe
* https://www.redhat.com/archives/libosinfo/2019-July/msg00026.html
* https://www.openwall.com/lists/oss-security/2019/07/08/3

Reply:
I think all that information is hold by NIST which is publishing the CVEs. Each CVE contains all related links, so in this case https://nvd.nist.gov/vuln/detail/CVE-2019-13313

Reply:
I guess we can link there then.

Reply:
Sure, I thought people will look it up from the patch ``name`` but a comment is probably handy.